checkov/terraform/parser.py

```diff
@@ -661,6 +662,24 @@ def _load_or_die_quietly(file: os.PathLike, parsing_errors: Dict,
         return None
 
 
+def _hcl2_load_with_timeout(f: io.TextIOWrapper) -> Dict:
+    # Start bar as a process
+    raw_data = None
+    q = Queue()  # used by the child process to return its result
+    p = Process(target=_hcl2_load, args=(q, f))
+    p.start()
+    raw_data = q.get(block=True, timeout=60)  # Wait until the file is parsed, up to 60 seconds. A timeout will raise a queue.Empty exception
+    p.join()  # Make sure the process is finished and closed
+    return raw_data
+
+
+def _hcl2_load(q: Queue, f: io.TextIOWrapper) -> None:
+    try:
+        raw_data = hcl2.load(f)
+        q.put(raw_data)
+    except Exception:
+        q.put(None)
```

Comment:
can you please log which file have failed and on what exception?

Reply:
+ are those added to the report parsing errors dictionary?

Reply:
Yes, they are added to the report